lib/pool.dart

```diff
@@ -36,12 +36,20 @@ class Pool {
   /// first regardless of what order the `onRelease` callbacks complete in.
   final _onReleaseCompleters = Queue<Completer<PoolResource>>();
 
-  /// The maximum number of resources that may be allocated at once.
   final int _maxAllocatedResources;
 
-  /// The number of resources that are currently allocated.
+  /// The maximum number of resources that may be allocated at once.
+  int get maxAllocatedResources => _maxAllocatedResources;
```

Comment:
You could just make `_maxAllocatedResources` public and avoid the property!

Reply:
makes sense, fixed